docs/stories/components/Checkbox.stories.js

```diff
@@ -25,21 +51,112 @@ function CheckboxStory() {
           value={val}
         />
       </div>
+      <div style={{ marginTop: '-5px', marginBottom: 20 }}>
+        <ul>
+          <Checkbox
+            name="selectAll"
+            onChange={handleChange}
+            message={message}
+            someChecked={hasSomeCheckboxSelected && !areAllCheckboxesSelected}
```

Comment:
This prop can't work because you use the `Checkbox` component from buffetjs/core but this prop is only available in the `Checkbox` component from buffetjs/styles. 
So the state doesn't exist.

Reply:
In fact, if you check out the code of the `Checkbox` component in `@buffetjs/core` you will see that we are passing all the props to the styled checkbox so it is working properly

Reply:
It didn't work the last time I tested but now it works!